dashboard/test/controllers/followers_controller_test.rb

```diff
@@ -94,6 +94,21 @@ class FollowersControllerTest < ActionController::TestCase
     assert_equal @chris_section, follower.section
   end
 
+  test "student_user_new errors when joining a section with deleted teacher" do
+    @laurel.update(deleted_at: Time.now)
```

Comment:
The variable name saddened me too. And though I thought about changing, I decided leaving it was a good way to remember Laurel.

Though I wasn't close to her (like you and others), nor did I know her for that long, Laurel had a surprising impact on me.